src/main/java/com/baidu/hugegraph/backend/store/tikv/TikvTable.java

```diff
@@ -92,6 +92,21 @@ public void clear(Session session) {
     @Override
     public void insert(Session session, BackendEntry entry) {
         assert !entry.columns().isEmpty();
+        if (entry.ttl() > 0L) {
+            for (BackendColumn col : entry.columns()) {
+                assert entry.belongToMe(col) : entry;
+                // Tikv ttl unit is seconds
+                session.put(this.table(), col.name, col.value,
+                            entry.ttl() / 1000);
+            }
+        } else {
+            for (BackendColumn col : entry.columns()) {
+                assert entry.belongToMe(col) : entry;
+                // Tikv ttl unit is seconds
```

Comment:
don't need to ttl comment since there is no ttl context

Reply:
I see our client ttl unit is millisecond， but tikv is seconds, so I comment it

Reply:
we can define `ttl = entry.ttl() / 1000` at line 94 and move the comment there

Reply:
ok